src/main/java/github/scarsz/discordsrv/listeners/DiscordChatListener.java

```diff
@@ -125,7 +135,7 @@ public void onGuildMessageReceived(GuildMessageReceivedEvent event) {
                         output = format.replace(placeholder, msg);
                     }
                     event.getAuthor().openPrivateChannel().queue(privateChannel ->
-                            privateChannel.sendMessage(output).queue());
+                            privateChannel.sendMessage(output).queue(null, new ErrorHandler().ignore(ErrorResponse.CANNOT_SEND_TO_USER)));
```

Comment:
Sure ignoring this exception is fine, but that is not a good solution to avoiding error responses when sending **invalid requests** (event.getAuthor() is a fake user when the message is from a webhook)